data/hai/hai news.txt

```diff
@@ -513,7 +513,7 @@ news "unfettered hai dialog"
 		word
 			"How did you get here? You should've been blasted to bits by our ships."
 			"I came here because this is the only place in the galaxy where I feel wanted."
-			"Weak human. Why do you dare to come to the land of the true Hai?"
+			"Weak human. Why do you dare to come to the land of the True Hai?"
```

Comment:
"true Hai" seems to be used more than "True Hai," so I don't think this change (or the others like it) are helpful.